impl/src/main/java/org/ehcache/impl/internal/store/AbstractValueHolder.java

```diff
@@ -29,6 +29,8 @@
  */
 public abstract class AbstractValueHolder<V> implements Store.ValueHolder<V> {
 
+  public static final TimeUnit TIME_UNIT = TimeUnit.MILLISECONDS;
```

Comment:
An abstract value holder cannot have a notion of default time unit; this can only be done in non-abstract implementations, i.e.: this should move to `ClusteredValueHolder`.